ex4-tema2.js

```diff
@@ -0,0 +1,18 @@
+//Returnează numărul de vocale dintr-un șir de caractere (string)
+//Creați o funcție care va returna un număr întreg al numărului de 
+//vocale găsite într-un șir. Aceasta este o modalitate excelentă de a practica determinarea caracteristicilor unui set de date.
+const str = "Acesta este un string";
+const countVowels = (str = "") => {
```

Comment:
Nu e nevoie de initializare variabilei. 
```suggestion
const countVowels = (str) => {
```